### DISCUSSION TITLE
Custom foreign keys for relations

### DISCUSSION THREAD

**Discussion by nettsite** [orchidsoftware/platform#1736] [Q&A] [answered]
I need two relationships between a licenses table and the users table, e.g. in the licenses table I need licensee_id and licensor_id, both as foreign keys to the id in the users table.

Eloquent relationships provide for this:

`public function user()
{
    return $this->belongsTo(User::class, 'foreign_key');
}
`

How can I achieve this within Orchid, and more specifically within the CRUD package?

**Comment by nettsite** [comment, marked as answer]
I scratched about a bit and found it:

`Relation::make('licensor_id')
                ->title('Licensor Name')
                ->fromModel(User::class, 'full_name','id'),`

I don't see it mentioned in the docs at https://orchid.software/en/docs/field/#relation

**Reply by MichelNZOLO** [reply]
Je galerais un peu mais grâce à toi, merci!
